src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -609,6 +610,59 @@ public void AssemblyResolve_RequestingAssembly()
         }
 
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/corefx/issues/22555", TargetFrameworkMonikers.Uap)]
```

Comment:
Also please have this test run inside the remote invoke. I prefer setting current culture in isolation in general.